TrackerGeom/src/Panel.cc

```diff
@@ -1,52 +1,63 @@
 //
 // Hold information about one Panel in a tracker.
 //
-//
-//
 // Original author Rob Kutschke
 //
 
-#include <sstream>
-
 #include "TrackerGeom/inc/Panel.hh"
+#include "cetlib_except/exception.h"
+#include <sstream>
 
 using namespace std;
+using CLHEP::HepRotation;
+using CLHEP::Hep3Vector;
 
 namespace mu2e {
+// define equivalence unique panels
+  StrawIdMask Panel::_sidmask(StrawIdMask::uniquepanel);
 
+// non-functional constructor needed by Proditions system: this should be eliminated FIXME!
```

Comment:
If you remove the default constructor of a Proditions object Proditions fails to link.  As far as I can tell this is because the Proditions service actually creates that object, then uses it to call the name() member, and then discards that object.  It is wasteful to create an object containing >20K elements (like Tracker) just to get a character string.  This comment is really just a reminder, I can remove it and make this an issue in github if you prefer.  I believe this is part of the 'threading the C++ needle' issue you brought up in an earlier thread, which we still need to discuss to find a cleaner solution.

Reply:
I have a fix for the name problem, I'll put it in after this is merged.